slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -51,11 +53,13 @@ static const int samplingFactor = 1;
 // Conversion to number of bytes used in colored PCD encoding
 static const int pixelBytetoPCDByte = 16 / 4;
 // Quaternion to rotate axes to the XZ plane
-static const Eigen::Quaterniond pcdRotation(0.7071068, 0.7071068, 0, 0);
+static const Eigen::Quaterniond pcdRotation(0.7071068, -0.7071068, 0, 0);
 // Static offset quaternion, so orientation matches physical intuition.
 // This will result in rotations occurring within the y axis to match 2D mapping
 // in the XZ plane
-static const Eigen::Quaterniond pcdOffsetRotation(0.7071068, -0.7071068, 0, 0);
+static const Eigen::Quaterniond pcdOffsetRotation(0.7071068, 0.7071068, 0, 0);
```

Comment:
Why is this changing?

Reply:
should have written at the top of the PR but additional fixes this PR looked to tackle are:

1. capturing the failed serialization state
2. a fix for the map appearing "flipped" when viewing, though I am only fixing this for the PCD endpoint.